docs/configuration.md

```diff
@@ -3,7 +3,7 @@
 Ruff can be configured through a `pyproject.toml` file, a `ruff.toml` file, or the command line.
 
 For a complete enumeration of the available configuration options, see
-[_Settings_](/docs/settings/).
```

Comment:
I think these are intentional because they're relative links once the docs are _live_.

Reply:
E.g., if you open. https://beta.ruff.rs/docs/configuration/, and click Settings, it takes you to the right place.

Reply:
Most Markdown documentation systems including MkDocs ([documentation](https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages)) expect you to write relative links to Markdown files: `[_Settings_](settings.md)`, `[_FAQ_](faq.md#ruff-tried-to-fix-something-but-it-broke-my-code)`. These are transformed at build time to point to the right HTML page.

Reply:
I'll fix this.

Reply:
@charliermarsh don't know if you're aware, but this is still broken, open https://github.com/charliermarsh/ruff/blob/main/docs/configuration.md and try to click `see Settings`

Reply:
I think this is expected though. These files are meant to be accessed via the docs (https://beta.ruff.rs/), rather than through GitHub.